frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -17,18 +17,20 @@ export interface NodeInfo {
 }
 
 export default class NodesApi extends ExplorerApi {
-  async getNodes(limit: number = 15) {
+  async getNodes(limit: number = 15, endTimestamp?: number) {
     try {
       return await this.call<NodeInfo[]>("select", [
         `SELECT ip_address as ipAddress, moniker, account_id as accountId, node_id as nodeId, signature, 
-                last_seen as lastSeen, last_height as lastHeight, last_hash as lastHash,
+                last_seen as timestamp, last_height as lastHeight, last_hash as lastHash,
```

Comment:
I suggest we don't break our APIs throughout the project just for the sake of `autoRefreshHandler`. Let's add a way to specify a "getter" function for `autoRefreshHandler`, which will extract the `timestamp` / `blockTimestamp` / `lastSeen` / or whatever it is from a given element. The same goes for `endTimestamp`, as from the semantics point of view, it does not make sense for `NodesApi.getNodes` to have `endTimestamp` argument.